src/config/blackboxTests/config.properties

```diff
@@ -1 +1 @@
-orderlyweb.api.url=http://orderly_web_web:8888/api/v1
+orderlyweb.api.url=http://orderly_web_web:8888/api/v2
```

Comment:
I guess this is going to need updating in montagu settings too?
https://github.com/vimc/montagu/tree/master/settings

Reply:
yes PR here https://github.com/vimc/montagu/pull/228